amundsen_application/static/webpack.dev.ts

```diff
@@ -1,4 +1,27 @@
+import * as path from 'path';
 import merge from 'webpack-merge';
-import commonConfig from './webpack.common'
+import commonConfig, {htmlWebpackPluginConfig} from './webpack.common';
+import MiniCssExtractPlugin from 'mini-css-extract-plugin';
 
-export default merge(commonConfig, {});
+export default merge(commonConfig, {
+  devServer: {
+      contentBase: path.join(__dirname, '../build/lib/amundsen_application/static/dist'),
+      publicPath: '/static/dist/',
+      watchContentBase: true,
+      port: 60480,
+      proxy: {
+          '!(/static/dist/**.*)': {
+              target: 'http://127.0.0.1:60477',
+              secure: false,
+          },
+      },
+  },
+  output: {
+    path: path.join(__dirname, '/dist'),
+    filename: '[name].js',
+  },
+  plugins: [
+      new MiniCssExtractPlugin(),
+      ...htmlWebpackPluginConfig("http://localhost:60480"), // should match port number above for devserver
```

Comment:
Does this mean folks will be accessing their local development version of the UI at `http://localhost:60480` instead of `http://localhost:5000`?

Reply:
Whoops! Internally we use different ports.